playbooks/templates/charts/argocd/argocd-values.yaml.j2

```diff
@@ -0,0 +1,2920 @@
+## Argo CD configuration
```

Comment:
Do we need to put the whole file here? I feel those are anyway defaults in 99% of cases

Reply:
will trim a bit